hls-writer/hls_writer.py

```diff
@@ -348,6 +370,7 @@ def hls_writer(layer_list, yamlConfig):
             newline += 'typedef {precision} accum_default_t;\n'.format(precision=yamlConfig["DefaultPrecision"])
             newline += 'typedef {precision} weight_default_t;\n'.format(precision=yamlConfig["DefaultPrecision"])
             newline += 'typedef {precision} bias_default_t;\n'.format(precision=yamlConfig["DefaultPrecision"])
+            #newline += 'typedef {precision} alpha_default_t;\n'.format(precision=yamlConfig["DefaultPrecision"])
```

Comment:
I have left this commented out instead of removing it because I wanted to discuss if using separate precision for alpha is worthwhile, but I am not sure if it makes sense, so I will remove it.

Reply:
Ok, understood.  Yeah, I think it's good the way it is.